src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -18,3 +19,7 @@ It remains possible to configure your own logging behavior by setting the `exclu
 In addition, the `SimpleMessageListenerContainer` consumer restart after such an exception is now logged at DEBUG level by default (previously INFO).
 A new method `logRestart()` has been added to the `ConditionalExceptionLogger` to allow this to be changed.
 See xref:amqp/receiving-messages/consumer-events.adoc[Consumer Events] and <<channel-close-logging>> for more information.
+
+==== `@AbstractConnectionFactory` Changes
+A new property `connectionCreatingBackOff` allows to create `SimpleConnection` with backoff policy to support for retry mechanism at the connection level for creating channels to avoid the exception in connection module
```

Comment:
The period in the end of sentence.

Can we also mentioned this new feature in the `connections.adoc`?
And add link to that chapter back into this new section.
Also, pay attention how all the sections in this `whats-new.adoc` have an id with `31` prefix.

Reply:
I've tried adding it, but I'm not sure if it meets the requirements.